cmake/setup_write_config.cmake

```diff
@@ -41,7 +41,8 @@ FILE(WRITE ${CMAKE_BINARY_DIR}/revision.log
 
 SET(_log_detailed "${CMAKE_BINARY_DIR}/detailed.log")
 SET(_log_summary  "${CMAKE_BINARY_DIR}/summary.log")
-FILE(REMOVE ${_log_detailed} ${_log_summary})
+SET(_log_feature "${CMAKE_BINARY_DIR}/deal.IIFeatureConfig.cmake")
```

Comment:
Please set to
`${CMAKE_BINARY_DIR}/${DEAL_II_PROJECT_CONFIG_RELDIR}/${DEAL_II_PROJECT_CONFIG_NAME}FeatureConfig.cmake`